extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/Http.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkus.vertx.http.runtime;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+
+import io.quarkus.runtime.annotations.ConfigDocIgnore;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.SmallRyeConfig;
+import io.smallrye.config.WithDefault;
+
+@ConfigMapping(prefix = "quarkus.http")
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public interface Http {
```

Comment:
Doing it this way means that we need to keep it in sync with the other HTTP config right?

I'm not entirely convinced it's going to be practical.

Reply:
No, it doesn't need to be in sync. It is possible to have multiple configuration mappings for the same properties, so we can have a "public" one that extensions and users can use at will and an internal one that can change.

Reply:
Except if you change the name of the property for the internal one, you will have to do the same here, given they have the exact same name? Or am I mistaken?

Or you will use a trick?